images/linux/scripts/installers/mysql.sh

```diff
@@ -44,7 +44,8 @@ echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "MySQL ($(mysql --version))"
 DocumentInstalledItem "MySQL Server (user:root password:root)"
 DocumentInstalledItem "MS SQL Server Client Tools"
+DocumentInstalledItem "MySQL service is disabled by default. Use the following command as a part of your job to start the service: `sudo systemctl start mysql.service`"
```

Comment:
You should use single quotes to escape backtick, because `sudo systemctl start mysql.service` -  is interpreted as a command.

_$ DocumentInstalledItem "MySQL service is disabled by default. Use the following command as a part of your job to start the service: `sudo systemctl start mysql.service`"_
```
System has not been booted with systemd as init system (PID 1). Can't operate.
- MySQL service is disabled by default. Use the following command as a part of your job to start the service:
```